src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj

```diff
@@ -28,6 +28,7 @@
     <RunCodeAnalysis>true</RunCodeAnalysis>
     <CodeAnalysisRuleSet>..\src.ruleset</CodeAnalysisRuleSet>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
Why do we need unsafe blocks? There should be safe APIs that you can use that don't require pointer use.